docs/user-guide/operations/performance-tuning-tips.md

```diff
@@ -0,0 +1,161 @@
+# Performance Tuning Tips
+
+A GreptimeDB instance's default configuration may not fit all use cases. It's important to tune the database configurations and usage according to the scenario.
+
+GreptimeDB provides various metrics to help monitor and troubleshoot performance issues. The official repository provides [Grafana dashboard templates](https://github.com/GreptimeTeam/greptimedb/tree/main/grafana) for both standalone and cluster modes.
+
+## Query
+
+### Metrics
+
+The following metrics help diagnose query performance issues:
+| Metric | Type | Description |
+|---|---|---|
+| greptime_mito_read_stage_elapsed_bucket | histogram | The elapsed time of different phases of a query in the storage engine. |
+| greptime_mito_cache_bytes | gauge | Size of cached contents |
+| greptime_mito_cache_hit | counter | Total count of cache hit |
+| greptime_mito_cache_miss | counter | Total count of cache miss |
+
+
+### Using cache for object stores
+
+While using object stores, it's highly recommended to enable the object store read cache and the write cache in the storage engine. This could reduce query time by more than 10 times.
```

Comment:
```suggestion
It's highly recommended to enable the object store read cache and the write cache in the storage engine. This could reduce query time by more than 10 times.
```

Users use object stores, there is no `while`.